platform/saivpp/vpplib/SwitchStateBaseAcl.cpp

```diff
@@ -597,65 +898,100 @@ sai_status_t SwitchStateBase::AclTblConfig(
 				meta->attridname);
 	    }
 	    
-	    if (attr->id == SAI_ACL_ENTRY_ATTR_FIELD_ACL_RANGE_TYPE) {
+        if(attr->id == SAI_ACL_ENTRY_ATTR_FIELD_TUNNEL_TERMINATED) {
```

Comment:
Collapse into 1 if as it's part of the same check.

Reply:
Updated